aws.go

```diff
@@ -129,9 +133,13 @@ func scrape(collector *cwCollector, ch chan<- prometheus.Metric) {
 				NextToken: nextToken,
 			})		
 			if err != nil {
+				collector.ErroneousRequests.Inc()
 				fmt.Println(err)
 				continue
 			}
+			else {
+				collector.SuccessfulRequests.Inc()
```

Comment:
Then, don't do `collector.ErroneousRequests.Inc()` or `collector.SuccessfulRequests.Inc() here any more. ( Leave the incrementing to the `scrapeSingleDataPoint()` function.

 ```suggestion

```

Reply:
Done :)